cli.mdx

```diff
@@ -0,0 +1,39 @@
+---
+name: CLI   
+route: /cli
+---
+
+# CLI - Command Line Interface
+
+Tool to help generate files from sdkgen to specific targets.
+
```

Comment:
You should mention the installation prerequisites and the how to:

- A current/LTS version of Node.js is required;
- It should be installed with `npm i -g @sdkgen/cli`

I'd also attempt to rewrite that line to add more information about what the CLI does.